attributes/default.rb

```diff
@@ -10,30 +10,33 @@
   default["sensu"]["windows"]["package_options"] = nil
   default["sensu"]["windows"]["install_dotnet"] = true
   default["sensu"]["windows"]["dotnet_major_version"] = 4
+  default["sensu"]["version"] = "1.0.2-1-x64"
+  default['sensu']['windows_package_version'] = '1.0.2.1'
```

Comment:
Is this required to use the new logrotate? I'd prefer to keep the changes as isolated as possible. I am not sure that we should default to 1.x yet as there are a handful of community plugins that have conflicts and need to be worked out first. You can always overwrite the version in your wrapper cookbook.

Reply:
This one is tricky.

Installing "1.0.2-1-x64" results in '1.0.2.1' in the registry. 

![afbeelding](https://user-images.githubusercontent.com/2706796/31534832-e45b2ebe-aff8-11e7-8165-0e816bb6370e.png)

As it does not has the 1.0.2-1-x64 display version it installs the client every chef run. This should imo be fixed in the MSI itself to keep the MSI version the same as the display version in the Windows registry. If that happens the attribute is absolete.

Reply:
ping @cwjohnston whos the best person to discuss the msi fix for version numbers?

Reply:
This `-x64` suffix may be an artifact introduced by the toolchain changes starting in Sensu 0.27. Opening an issue against https://github.com/sensu/sensu-omnibus would help us track and fix this.